openerp/openupgrade/doc/source/development.rst

```diff
@@ -1,7 +1,43 @@
+=========================================================
 Migration script development
-++++++++++++++++++++++++++++
+=========================================================
 
 .. toctree::
    :maxdepth: 2
 
    devfaq
+
```

Comment:
Index does not make any sense anymore. Please move your added lines to a separate file and refer to that in this index instead of adding it here.